tests/allure_pytest_bdd/acceptance/allure_tags_test.py

```diff
@@ -0,0 +1,203 @@
+    (
+        "suites_labels",
+        """
+        @allure.suite('My Suite')
+        @allure.parent_suite('My Parent Suite')
+        @allure.sub_suite('My Sub Suite')
+        """.strip(),
+        (
+            has_parent_suite('My Parent Suite'),
+            has_suite('My Suite'),
+            has_sub_suite('My Sub Suite'),
+        ),
+    ),
+    (
+        "bdd_labels",
+        """
+        @allure.epic('My epic')
+        @allure.feature('My feature')
+        @allure.story('My story')
+        @allure.description('My description')
+        """.strip(),
+        (
+            has_feature("Basic allure-pytest-bdd usage"),  # from the scenario
+            has_epic("My epic"),
+            has_feature("My feature"),
+            has_story("My story"),
+            has_description("My description"),
+        ),
+    ),
+    (
+        "bdd_multiply_labels",
+        """
+        @allure.epic('My epic 1')
+        @allure.epic('My epic 2')
+        @allure.feature('My feature 2')
+        @allure.feature('My feature 3')
+        @allure.story('My story 1')
+        @allure.story('My story 2')
+        """.strip(),
+        (
+            has_epic("My epic 1"),
+            has_epic("My epic 2"),
+            has_feature("Basic allure-pytest-bdd usage"),
+            has_feature("My feature 2"),
+            has_feature("My feature 3"),
+            has_story("My story 1"),
+            has_story("My story 2"),
+        ),
+    ),
+    (
+        "id_label",
+        """
+        @allure.id(123)
+        """.strip(),
+        (
+            has_label("as_id", 123),
+        ),
+    ),
+    (
+        "severity_label",
+        """
+        @allure.severity('critical')
+        """.strip(),
+        (
+            has_severity("critical"),
+        ),
+    ),
+    (
+        "manual_label",
+        """
+        @allure.manual
+        """.strip(),
+        (
+            has_label("ALLURE_MANUAL", True),
+        ),
+    ),
+    (
+        "tags",
+        """
+        @pytest.mark.cool
+        @pytest.mark.stuff
+        @allure.tag('foo')
+        """.strip(),
+        (
+            has_tag("cool"),
+            has_tag("stuff"),
+            has_tag("foo"),
+        ),
+    ),
+    (
+        "custom_labels",  # !!!
+        """
+        @allure.label("Application", "desktop", "mobile")
+        """.strip(),
+        (
+            has_label("Application", "desktop"),
+            has_label("Application", "mobile"),
+        ),
+    ),
+    (
+        "links",
+        """
+        @allure.link('https://example.org/simple-link')
+        @allure.issue('https://example.org/issue')
+        @allure.testcase('https://example.org/testcase')
+        """.strip(),
+        (
+            has_link('https://example.org/simple-link'),
+            has_issue_link('https://example.org/issue'),
+            has_test_case_link('https://example.org/testcase'),
+        ),
+    ),
+])
+def test_simple_passed_scenario_with_allure_tags(
```

Comment:
Let's rename the file to `allure_decorators_test.py` and this test function to `test_simple_passed_scenario_with_allure_decorators`. Tags are just a part of what's tested here (other parts are descriptions, links, other labels, etc).

```suggestion
def test_simple_passed_scenario_with_allure_decorators(
```

Reply:
@delatrie Hello! 
Thank you so much. I fixed your comments.

Reply:
@delatrie Hello! Do you have an estimate when you will release a new version with changes?